fluentd-forward/state.go

```diff
@@ -0,0 +1,198 @@
+/*
+Copyright 2015-2021 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"net"
+	"strings"
+	"time"
+
+	"github.com/gravitational/trace"
+	"github.com/peterbourgon/diskv"
+	log "github.com/sirupsen/logrus"
+)
+
+const (
+	// cacheSizeMax max memory cache
+	cacheSizeMax = 1024
```

Comment:
This name should include a unit

Reply:
Renamed to `cacheSizeMaxBytes`. I think, bytes might be assumed by default.

Reply:
Should be named `cacheSizeMaxKiloBytes` I think. From what I gather from the docs it assumes kilobytes.

Reply:
I might be mistaken, but according to docs looks like it's bytes: `CacheSizeMax uint64 // bytes`